vignettes/including-static-dashboards.Rmd

```diff
@@ -0,0 +1,107 @@
+---
+title: "Including Static Dashboards in Crunch"
+description: "How to include Flexdashboards in a Shiny App"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Static Dashboards}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+Crunch allows you to host Shiny apps on the Crunch Jupyter server. Shiny apps are great, but they can be difficult to program, and can be very slow if not designed properly. [Flexdashboards](https://rmarkdown.rstudio.com/flexdashboard/) provide an easy-to-use alternative to Shiny apps. Instead of re-rendering dynamically based on the user action, Flexdashboards are effectively static websites which only include data which is live at the time of rendering. This can be desirable both because they are much easier to write, and because you can test and validate the dashboard's behavior before it goes to the client. Additionally, since you have finer control over when a dashboard is rendered, you can include more computationally expensive operations like fitting a machine learning model without affecting user experience.
+
+You can host Flexdashboards on Crunch's Shiny infrastructure using Shiny's `includeHTML` function. What `includeHTML` does is add an HTML file to the shiny UI. If all you include in the shiny UI is the Flexdashboard html file, then the Shiny app effectively becomes a static site. 
+
+### Basic workflow
+Follow these steps to include a Flexdashboard in your Shiny app: 
+1) First create a new Rmarkdown document with a Flexdashboard output option. 
+1) Render the Rmd file into an html file
+1) Uploaded the rendered html file to your app's Jupyter folder
+1) In your shiny UI code add `includeHTML(<filename>)` to the `ui` object. Your `app.R` file might look something like this:
+
+```
+library(crunchy)
+ui <- fluidPage(
+    includeHTML("dashboard")
+)
+
+server <- function(input, output, session) {}
+
+# Run the application 
+shinyApp(ui = ui, server = server)
+```
+Note that we haven't included any authentication in this app which means that users who do not have access to a dataset could theoretically access it. The next step is to check that the user has access to Crunch and to display dashboards to users based on which company they are from.  
+
+### Conditional dashboards and user identification
```

Comment:
Can you separate these two issues? Add a simpler version between the previous example and the following one: an example that just checks auth and then serves the dashboard. Then add the conditional user behavior after.